aiohttp_session/redis_storage.py

```diff
@@ -43,13 +43,14 @@ def __init__(
             decoder=decoder,
         )
         if aioredis is None:
-            raise RuntimeError("Please install aioredis")
+            raise RuntimeError("Please install redis")
         # May have installed aioredis separately (without aiohttp-session[aioredis]).
-        if StrictVersion(aioredis.__version__).version < (2, 0):
-            raise RuntimeError("aioredis<2.0 is not supported")
+        lib_version = tuple(map(int, redis.__version__.split('.')[:2]))  # type: ignore
+        if lib_version < (4, 3):
```

Comment:
Appears to already be a `redis.VERSION`, so there's no need to duplicate the tuple code here.

Reply:
Also, adding to stubs: https://github.com/python/typeshed/pull/7838